.github/workflows/tests.yaml

```diff
@@ -0,0 +1,15 @@
+name: Docker Compose Actions Workflow
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup the environment
+        run: docker-compose -f dev/docker-compose.yaml  -f dev/docker-compose.dev.yaml up -d
+      - name: Run busted tests
+        run: docker-compose -f dev/docker-compose.yaml  -f dev/docker-compose.dev.yaml exec openresty busted
```

Comment:
I also noticed that the tests for sasl have issues with loading the module.

https://github.com/Kong/lua-resty-kafka/blob/master/.busted#L5

seems wrong